card/VERSION

```diff
@@ -1 +1 @@
-1.14.8s
+1.15.pre
```

Comment:
> The name of the repo base is still something we should be in sync on because it is needed in the tools.

I don't think I follow.  What tools?  Like rake tasks?

Basically, I think it's fair to assume that core developers will be working from a local copy of the gem, so any code intended for core developers (including building the gem from the gemspec file) can safely refer to the repo base.

Any other code, however (code that does stuff for wagneers or developers) should NOT refer to the repo base, because the base usually won't be there at all.

Agree that there could be command issues, but they should be easy enough. to fix.  We could also make it so that, for example, Wagn::Version gets its methods from Card::Version one way or another.

There is a version spec, yes, and at one point it was working locally (where the repo base existed) but not on semaphore (where it didn't).  That's how I got clear on the need to have it somewhere in a gem dir ;)